### ISSUE TITLE
[Resource creation] guide on convincing leadership to fund OSS

### ISSUE THREAD

**Issue by juliaferraioli** [fossfunders/fossfunders.com#1]
It would be great to have a page / guide on different strategies to make the case for funding open source, similar to how some conferences have "convince your boss to send you here" page. We could outline different angles and arguments that target the needs of different types of companies.

**Comment by DuaneOBrien** [comment]
We had a Making The Case For A FOSS Fund slide deck at one point, but we may not have access to it any longer.

**Comment by ShaneCurcuru** [comment]
Late to the party here, but this is absolutely the next big critical thing to have.  Non-FOSS thinking budget holders aren't going to be impressed by the website listing, unless they have a simple-to-them description of the ROI *they* feel they can get by supporting FOSS.

Aren't there a handful of other "benefits of FOSS to companies (besides shared maintenance)" that we could start from?

**Comment by chadwhitacre** [comment]
I disagree. My view is that there are enough companies out there that are giving in some form or fashion that pulling them all together in one place will create something new: social pressure via advocacy from devs/talent. When interviewing for jobs, devs will ask, "Why is your company not listed on fossfunders.com?" This will be how we chip away at less-eager companies and reach the tipping point, eventually sweeping through the majority of companies over the next 5 to 10 years. Better to own the reality that Open Source economics is IOR, not ROI, and work with social pressure (and taxation, but that's not what this group is about), than try to manufacture a pain point that isn't there.

Look how fast every company became an AI company. Companies are highly susceptible to social validation. We can do this! :-)

**Comment by anehzat** [comment]
@ShaneCurcuru  A lot of people talk about sustainability of open source & security concerns from developer burnouts which result in code being left dormant. This talk by Bret working on dormant code is the strongest value proposition / ROI I've seen to date https://www.youtube.com/watch?v=e5FV-AnKPlo If a company cares about their code quality & open source security, it would be negligent for them to ignore this point.

**Comment by DuaneOBrien** [comment]
@chadwhitacre I don't see "create social pressure" and "provide supporting materials" as mutually exclusive.

**Comment by chadwhitacre** [comment]
Sure, but if the supporting materials focus too strongly on ROI demonstration, then they reinforce the wrong narrative. The reality is that companies are essentially walking away from the restaurant without paying the bill. That's the best image I've found so far for the Open Source funding situation, and something along those lines should be our overall framing. If we offer "make the ROI case" materials, they should be couched as tactical concessions, not as our overarching strategy.
